lib/helpers/is-interactive-element.js

```diff
@@ -39,10 +41,6 @@ function isAudioVideoWithControls(node) {
   return (node.tag === 'audio' || node.tag === 'video') && ast.hasAttribute(node, 'controls');
 }
 
-function isHyperLink(node) {
-  return node.tag === 'a' && ast.hasAttribute(node, 'href');
-}
```

Comment:
See above, I think this needs to come back.

Reply:
see response above.